### PR TITLE
redesign/component/menu

### PR DESCRIPTION
**Pull request by sujoydev99**

### CODE REVIEW

**Comment by ShahRohan27** [comment]
@sujoydev99 please revert the package.json changes

**Comment by sujoydev99** [comment]
changes reverted

**Comment by sujoydev99** [comment]
@ShahRohan27 changes reverted

File: src/components/Menu/Menu.stories.tsx [hunk 1]
```diff
@@ -1,31 +1,299 @@
...
 import "./Menu.scss";
+import Button from "../Button";
 
-// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+const stories = {
+  simple: [
+    {
+      label: "Apple",
+      prefix: (
+        <svg fill="none" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```
Comment (ShahRohan27) [inline src/components/Menu/Menu.stories.tsx:12]: Refer checkbox story on how to pass Icons.

File: src/components/Menu/Menu.test.tsx [hunk 2]
```diff
@@ -1,58 +1,411 @@
...
-    userEvent.click(testImage);
-    const menuItemM2 = screen.getByText("M2");
-    userEvent.click(menuItemM2);
-    expect(selectedMenu).toBe("M2");
+    const menuBlock = getByTestId("n-menu-block");
+    const button = getByTestId("button");
+    expect(menuBlock.className).toContain("n-menu-block");
+    expect(menuBlock.className).toContain("n-menu-block-open");
+
+    //open
```
Comment (ShahRohan27) [inline src/components/Menu/Menu.test.tsx:404]: Remove if not required

File: src/base/variable.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @import "./fonts.scss";
 
 $BaseFontSize: 10px;
+$MenuItemFontSize: 16px;
```
Comment (ShahRohan27) [inline src/base/variable.scss:4]: Use $BaseFontSize + 6 instead of add a new variable

**Review by ShahRohan27 — CHANGES_REQUESTED** [review]

**Comment by sujoydev99** [comment]
Issue : -
Now : Redesigned menu component, captured all use cases.

# Menu Component
- simple
image


- only labels
Screenshot 2023-02-01 at 4 46 01 AM

- with links
Screenshot 2023-02-01 at 5 29 12 AM


- mixed
Screenshot 2023-02-01 at 5 30 03 AM


- no icons
Screenshot 2023-02-01 at 4 45 50 AM


- anchored to any element
Screenshot 2023-02-01 at 4 46 13 AM 
image


## tests
Screenshot 2023-02-01 at 5 38 53 AM

**Pull request merged**